erde/lib.lua

```diff
@@ -190,11 +190,14 @@ local function __erde_internal_load_source__(source, options)
   return loader()
 end
 
+local function source_loader_wrapper(...)
+  return ...
+end
```

Comment:
huh, this is a clever trick! I tried to apply it to the other loaders as well but cant seem to get it to work with everything. In any case for now this is fine, since I have plans to rework and improve (and make REAL tests) the somewhat awkward source loading / error rewriting currently happening